### DISCUSSION TITLE
Syncs Django user model with Supabase users.

### DISCUSSION THREAD

**Discussion by Prashant4900** [supabase/supabase#3185] [Questions]
how should we sync our Django users with supabase users so each user we created in Django is stored in supabase users?
because in general if we created a new user in Django it's rather than user supabase users it's created our own new table ( users ) database. so how should we avoid this?

**Comment by kangmingtay** [comment]
Hey @Prashant4900, 

> in general if we created a new user in Django it's rather than user supabase users it's created our own new table ( users ) database.

Could you elaborate more on this point?

Generally, I would recommend to first switch django's password hashing algorithm to use `bcrypt` instead of the default `PBKDF2`. If you're looking to migrate your django users over to supabase auth, you'll need to compare the schema difference between your users table in django and the `auth.users` table in Supabase and map the appropriate fields accordingly.

**Comment by biomassives** [comment]
@Prashant4900   I like your goal of having users synced on both systems.  Each db could trigger hooks to update the other i would think.  Did you arrive at a solution?
